themes/docsy/layouts/partials/community_links.html

```diff
@@ -7,6 +7,8 @@ <h2>Learn and Connect</h2>
 {{ with index $links "user"}}
 {{ template "community-links-list"  . }}
 {{ end }}
+<h2>Need More Than Community Support?</h2>
+<p>If you require a higher level of support, enterprise support is <a href="https://www.redhat.com/en/technologies/cloud-computing/openshift/what-is-openshift-service-mesh">provided by Red Hat.</a></p>
```

Comment:
I don't love this on the Community page, it sort of stinks up the whole community thing.  Also, not sure that page gets hit very much.  How about at the top of the FAQ page, perhaps below the FAQ title and above the actual FAQ listings?  That way people will see it when they actually go to a FAQ looking for a solution to a problem?

Reply:
changed - see the FAQ page now: https://deploy-preview-724--kiali.netlify.app/docs/faq/